prow/plugins/retitle/retitle_test.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package retitle
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/diff"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/github/fakegithub"
+)
+
+func TestHandleGenericComment(t *testing.T) {
+	var testCases = []struct {
+		name            string
+		state           string
+		action          github.GenericCommentEventAction
+		isPr            bool
+		body            string
+		trusted         func(string) (bool, error)
+		expectedTitle   string
+		expectedErr     bool
+		expectedComment string
+	}{
+		{
+			name:  "comment on closed issue is ignored",
```

Comment:
Is this test case emulating an otherwise-valid comment on a closed issue? Shouldn't action be set to github.GenericDommentActionCreated, with a body of "/retest foobar"?

Reply:
done